wandb/cli/cli.py

```diff
@@ -824,12 +824,12 @@ def _clean():
         _summary()
 
 
-@cli.command(context_settings=CONTEXT, help="Create a sweep")
-@click.option("--project", "-p", default=None, help="The project of the sweep.")
-@click.option("--entity", "-e", default=None, help="The entity scope for the project.")
+@cli.command(context_settings=CONTEXT, help="Initialize a hyperparameter sweep. Search for hyperparameters that optimizes a cost function of a machine learning model by testing various combinations.")
+@click.option("--project", "-p", default=None, help="""The name of the project where W&B runs created from the sweep are sent to. If the project is not specified, the run is sent to a project labeled Uncategorized.""")
```

Comment:
why do you need additional `""`?

Reply:
Good question, I added them because I figured that (eventually) we'd want to format it so it's not just one long line...